jobrunner/docker.py

```diff
@@ -37,22 +44,29 @@ class DockerTimeoutError(Exception):
     pass
 
 
+def docker(docker_args, timeout=DEFAULT_TIMEOUT, **kwargs):
+    args = ["docker"] + docker_args
+    try:
+        return subprocess_run(args, timeout=timeout, **kwargs)
+    except subprocess.TimeoutExpired as e:
+        raise DockerTimeoutError from e
```

Comment:
Separate to the timeout change, this is a nice little abstraction for the reader, all the calls being explicitly `docker()` calls below is great 👍

Reply:
Yeah, wish I'd done this in the first place and then adding the timeout would have been a lot easier!